gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "JWT Claim Check",
+  "summary": "Allow or deny traffic based on a JWT claim",
+  "description": [
+    "This Policy allow to block traffic based on a JWT token.",
+    "To verify any JWT claim can be used and can be compared ",
+    "using plain or liquid filters."
+  ],
+  "version": "builtin",
+  "configuration": {
+    "definitions": {
+      "value_type": {
+        "$id": "#/definitions/value_type",
+        "type": "string",
+        "oneOf": [{
+          "enum": ["plain"],
+          "title": "Evaluate 'value' as plain text."
```

Comment:
This assumes that the field using this is called `value` which is not true for all the cases in this schema. There's one called `resource`.  We should probably change the description to "Evaluate as plain text."

Reply:
good point, thanks!